ibis/expr/types/structs.py

```diff
@@ -112,6 +112,56 @@ def fields(self) -> Mapping[str, dt.DataType]:
         """Return a mapping from field name to field type of the struct."""
         return util.frozendict(self.type().pairs)
 
+    def unpack(self) -> ir.Table:
+        """Project the fields of `self` into its child table.
```

Comment:
Terminology question: I would have used called this it's parent table - the fields from inside the struct are projected out into the table containing the struct (the "parent" table). Is that an incorrect view of this term in ibis?

Reply:
The root of the expression tree is the output of calling `unpack` in this case. An input to that output is a child of it:

```mermaid
graph BT;
    Table-->Unpack;
```

Similar to arithmetic:

```mermaid
graph BT;
	1-->+;
	2-->+;
```

Reply:
Sure, but that's at the expression level, which I *think* an ibis user reading this docstring shouldn't care about? Rather what they care about is the effect on the data/tables of the operation. Since `unpack()` unpacks from a struct column into the containing table, that's what I think the docstring here should be describing. Not sure how best to phrase that though.

Reply:
> Sure, but that's at the expression level, which I think an ibis user reading this docstring shouldn't care about?

I'm not sure I follow how that would change the terminology used. "Expression level" has no bearing on the tree structure of the object exposed to the user.